examples/docker2singularity/docker2singularity.sh

```diff
@@ -85,37 +87,69 @@ echo "Size: $size MB for the singularity container"
 ################################################################################
 ### IMAGE CREATION #############################################################
 ################################################################################
-
+TMPDIR=$(mktemp -u -d)
+mkdir -p $TMPDIR
 
 creation_date=`echo ${creation_date} | cut -c1-10`
 new_container_name=$image_name-$creation_date.img
 $SUDOCMD singularity create -s $size $new_container_name
 $SUDOCMD docker export $container_id | $SUDOCMD singularity import $new_container_name
-$SUDOCMD docker inspect $container_id >> singularity.json
-sudo singularity copy $new_container_name singularity.json /
+$SUDOCMD docker inspect $container_id >> $TMPDIR/singularity.json
+sudo singularity copy $new_container_name $TMPDIR/singularity.json /
 
 # Merge the /etc/group file
-$SUDOCMD docker cp $container_id:/etc/group grouphost
-sort /etc/group grouphost | uniq -u > group
-$SUDOCMD singularity copy $new_container_name group /etc/group
-rm singularity.json
-$SUDOCMD rm grouphost
-$SUDOCMD rm group
+$SUDOCMD docker cp $container_id:/etc/group $TMPDIR/grouphost
+sort /etc/group $TMPDIR/grouphost | uniq -u > $TMPDIR/group
+$SUDOCMD singularity copy $new_container_name $TMPDIR/group /etc/group
+$SUDOCMD singularity bootstrap $new_container_name
```

Comment:
could you explain what you are trying to do here with bootstrap? The context I've seen it used in is to add some definition fine to an image that was just created, for example:

      sudo singularity bootstrap $new_container_name debian.def

but without a parameter after I think we are just going to get an error. Is it that you want to have some kind of optional argument for (also) including a bootstrap file?